### PR TITLE
No up event

### PR DESCRIPTION
**Pull request by raphaelpierquin**

### CODE REVIEW

File: public/paysagerenderer.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
-var io = io.connect();
 var canvas = {};
 var sketch = {};
 
 // get the playground id from a data-attribute generated by the view. Hacky.
```
Comment (etienneCharignon) [inline public/paysagerenderer.js]: Couldn't we inline thoses two variables? (and remove the comment)

File: public/programmerjs/receptiontransmission.js [hunk 2]
```diff
@@ -6,7 +6,13 @@ var Paysage = Paysage || {};
   // Require a sourcebuilder script defining Paysage.getCompleteCodeObject()
   // Require a codeinitialization script defining Paysage.setCodeId()
 
-  io = io.connect();
+  var playgroundid = document.getElementById('playgroundid').value;
```
Comment (etienneCharignon) [inline public/programmerjs/receptiontransmission.js]: here too, I would have inlined those variables

**Comment by raphaelpierquin** [comment]
Ready for review.

File: e2etests/tests/test.js [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
     browser.end();
+  },
+  'Code object is listed': function (browser) {
+    browser.page.programmer()
+      .navigate()
+      .setCodeId('toto')
+      .click('@go-live')
+      .waitForElementPresent('#objects li a:first-of-type', 2000, true, function(result) {
+        this.assert.containsText('#objects li a:first-of-type','toto');
+      });
```
Comment (etienneCharignon) [inline e2etests/tests/test.js]: You could also use the expect API as follow :
```
browser.page.programmer().expect
      .element('#objects li a:first-of-type').text.to.equal('toto');
```
This example is using a default timeout.

File: e2etests/tests/test.js [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+    browser.end();
```
Comment (etienneCharignon) [inline e2etests/tests/test.js]: For better performance, you'd better end the browser in the after function
```
  after : function(browser) {
    browser.end();
  },
```

File: e2etests/tests/test.js (hunk 3 above)
Comment (raphaelpierquin) [inline e2etests/tests/test.js]: nice. thanks !

File: e2etests/tests/test.js (hunk 4 above)
Comment (raphaelpierquin) [inline e2etests/tests/test.js]: Indeed, it reduces e2e tests execution time from 6s to 5.3s on my laptop.

File: public/programmerjs/receptiontransmission.js [hunk 5]
```diff
@@ -16,52 +22,42 @@ var Paysage = Paysage || {};
     Paysage.getCompleteCodeObject(emitData);
```
Comment (jonathanperret) [inline public/programmerjs/receptiontransmission.js]: Please make sure none of the two implementations of `getCompleteCodeObject` send a `playgroundId` anymore.

**Comment by jonathanperret** [comment]
Great!

**Pull request merged**